package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@podium/layout",
-  "version": "5.0.9",
+  "version": "5.0.8",
```

Comment:
Looks like some stuff got dropped in a merge, the version shouldn't be changed

Reply:
I messed up the merge a lil bit. But 5.0.9 should be the correct version.